crates/node/src/service.rs

```diff
@@ -442,8 +442,8 @@ pub fn new_full(
                 Arc::new(CelestiaClient::try_from(celestia_conf).map_err(|e| ServiceError::Other(e.to_string()))?)
             }
             DaLayer::Ethereum => {
-                let ethereum_conf = EthereumConfig::try_from(&da_path)?;
-                Arc::new(EthereumClient::try_from(ethereum_conf)?)
+                let ethereum_conf = EthereumConfig::try_from(&da_path).map_err(|e| e.to_string())?;
```

Comment:
Sill a few `to_string` there

Reply:
yes but the `new_full` function returns a `Result<TaskManager, ServiceError>` type so the `DaError` has to be converted (unless the ServiceError type includes DaError type)

Reply:
Oh sure sorry